### ISSUE TITLE
Element overlapping occurs when UIO text size is increased

### ISSUE THREAD

**Issue by BlueSlug** [fluid-project/handbook.floeproject.org#57]
## Describe the bug

As reported by @cindyli: "Gradually increase UIO font size, you will notice a few issues:

On the desktop view:

1. At some point, the menu bar at the top right will overlap with the logo at the top left;
2. Go to a page having the side menu, such as the "Perspectives" page, the main content overflows and stretches out of the screen width. The content in the left panel even gets cut off and loses some content.

On the mobile view:

1. The menu icon doesn't overlap with the logo but the full width of these two elements exceeds the screen size causing the overflow.
2. The page title, "Perspectives" in this case, does wrap into multiple lines. But these lines start to overlap on each other.
 
Instead of using UIO, if you use "Command" and "+" to increase font size, the browser will handle the font change quite nicely: the browser translates the font change to the viewport size change so that when the font size is large enough, the viewport will be switched from the desktop view to the mobile view. UIO doesn't have this ability yet.

WeCount site ran into the same issue. The decision was to remove the text size change from UIO."

### To reproduce

1. Go to the Perspectives page: https://handbook.floeproject.org/perspectives/
2. Click on "show preferences" to open the UIO panel
3. Increase the text size preference to 2
4. Reduce the viewport/window width
5. See error

### Expected behavior

Text should not be overlapping itself nor other elements on the page such as the header banner.

## Screenshots
<img width="1396" alt="157681330-5151064a-2ca5-425b-8c0d-e2017b2b1efc" src="https://user-images.githubusercontent.com/547933/158275158-bf6c7497-e82e-40cf-8738-5ca73548b458.png">

<img width="446" alt="157681900-3f695dd3-64f3-4025-93bc-af86e8000daa" src="https://user-images.githubusercontent.com/547933/158275160-a8cf9dab-bf53-4c90-9ba6-0b51299b2ade.png">

### Desktop

- OS: Windows 10 Pro 21H2
- Browser & version: Chrome 99

### Mobile device

- Device: Google Pixel 5
- OS: Android 12
- Browser & version: Chrome 99

### Assistive technology used

N/A

## Additional context or notes

As reported in #48 review
